### PR TITLE
doc(autoinstall): Remove incorrect statements, be more direct

### PR DESCRIPTION
**Pull request by holmanb**
## Commit message
```
doc(autoinstall): Remove incorrect statements, be more direct
```

## Additional Context
Top level keys are no longer ignored, so drop this incorrect statement.

Tailor this section more directly toward subiquity / autoinstall, since
that is where the frequently asked questions come from.

### CODE REVIEW

File: doc/rtd/reference/faq.rst [hunk 1]
```diff
@@ -15,21 +15,15 @@ Having trouble? We would like to help!
 - Find a bug? Check out the :ref:`reporting_bugs` topic to find out how to
   report one
 
-``autoinstall``, ``preruncmd``, ``postruncmd``
-==============================================
-
-Since ``cloud-init`` ignores top level user data ``cloud-config`` keys, other
-projects such as `Juju`_ and `Subiquity autoinstaller`_ use a YAML-formatted
```
Comment (aciba90) [inline doc/rtd/reference/faq.rst:22]: As we are removing all the references to Juju in this page, could we also remove the references to them, please?

* https://github.com/canonical/cloud-init/blob/8ceae8b7b504960e348ecda36c26db0b6fa12890/doc/rtd/reference/faq.rst?plain=1#L86
* https://github.com/canonical/cloud-init/blob/8ceae8b7b504960e348ecda36c26db0b6fa12890/doc/rtd/reference/faq.rst?plain=1#L109

**Review by aciba90 — APPROVED** [review]
LGTM, thanks! I left a minor nit comment.

File: doc/rtd/reference/faq.rst [hunk 2]
```diff
@@ -15,21 +15,15 @@ Having trouble? We would like to help!
...
-`discourse page`_.
+``autoinstall``
+===============
+
+Other projects, such as `Subiquity autoinstaller`_, use cloud-init to implement
+a subset of their features and have a YAML configuration format which combines
+``cloud-init``'s cloud-config with additional keys.
+
+If you are an autoinstall user, please direct questions to their IRC channel
+(``#ubuntu-server`` on Libera) or Discourse.
```
Comment (s-makin) [inline doc/rtd/reference/faq.rst:26]: Can we include a link to their Discourse?

**Review by s-makin — APPROVED** [review]
Apart from the mentioned comments, LGTM :)

**Pull request merged**